src/Files.App/Helpers/Application/AppLifecycleHelper.cs

```diff
@@ -265,14 +267,6 @@ public static void HandleAppUnhandledException(Exception? ex, bool showToastNoti
 				ex.Data[Mechanism.HandledKey] = false;
 				ex.Data[Mechanism.MechanismKey] = "Application.UnhandledException";
 
-				SentrySdk.CaptureException(ex, scope =>
-				{
-					scope.User.Id = generalSettingsService.UserId;
-				});
-
-
-				SentrySdk.CaptureException(ex);
-
```

Comment:
@yaira2 is there any reason to capture the exception twice?

Reply:
No, this was an oversight.